controllers/storagecluster/cephcluster.go

```diff
@@ -398,16 +398,27 @@ func newCephCluster(sc *ocsv1.StorageCluster, cephImage string, nodeCount int, s
 			},
 			Resources: newCephDaemonResources(sc),
 			ContinueUpgradeAfterChecksEvenIfNotHealthy: true,
-			LogCollector: rookCephv1.LogCollectorSpec{
-				Enabled:     true,
-				Periodicity: "24h",
-			},
+			LogCollector: sc.Spec.LogCollector,
 			Labels: rookCephv1.LabelsSpec{
 				rookCephv1.KeyMonitoring: getCephClusterMonitoringLabels(*sc),
 			},
 		},
 	}
 
+	// if logCollector is disabled use default values
+	if !cephCluster.Spec.LogCollector.Enabled {
+		cephCluster.Spec.LogCollector = rookCephv1.LogCollectorSpec{
+			Enabled:     true,
+			Periodicity: "daily",
+			MaxLogSize:  resource.NewQuantity(500000000, resource.BinarySI),
```

Comment:
Could you specify the size with "500Mi" instead of the bytes?

Reply:
resource.NewQuantity takes `int64` as a first arg and we have `resource.NewQuantity` only because we need the pointer return.

Reply:
It's convenient that it returns the pointer directly, but I worry about accidentally setting the wrong value since there are so many zeros. How about `ParseQuantity("500Gi")`?

Reply:
done.